### PR TITLE
docs: [IAC-1830]: IACM release notes (versions 0.175.0, 0.176.0 & 0.178.0)

### PR DESCRIPTION
**Pull request by richardblack-Harness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Back filling release for IACM into March 2024
* JIRA: [IAC-1830](https://harness.atlassian.net/browse/IAC-1830)

### Screenshots:
Screenshot 2024-04-24 at 15 47 00

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [x] Release notes/new features docs: Feature/version released to at least one prod environment.

[IAC-1830]: https://harness.atlassian.net/browse/IAC-1830?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: release-notes/infrastucture-as-code-management.md [hunk 1]
```diff
@@ -28,4 +28,21 @@ These release notes describe recent changes to Harness Infrastructure as Code Ma
...
   - **Enhanced Layout:** Increased spacing around store cards to prevent overlap of check marks and enhance readability and accessibility. (IAC-1601)
   - **Sensitive Field Icons:** The eye/hide icon is now exclusively used with sensitive fields to improve privacy and clarity. (IAC-1694)
----
\ No newline at end of file
+---
+
+## March 2024
+### Version: 0.178.0
+<!-- Released on: 25 March 2024 -->
+#### New features and enhancements:
```
Comment (brian-f-harness) [inline release-notes/infrastucture-as-code-management.md:36]:
```suggestion
#### New features and enhancements
```

File: release-notes/infrastucture-as-code-management.md [hunk 2]
```diff
@@ -28,4 +28,21 @@ These release notes describe recent changes to Harness Infrastructure as Code Ma
...
+  - **Connector Validation Enhancement:** We've implemented a validation check for connectors when creating or editing workspaces. This update ensures that all connectors used are currently available and of the correct type before proceeding with the operation, to help prevent errors related to non-existent or incorrect type connectors, enhancing the reliability and efficiency of workspace management. (IAC-1602)
+
+### Version: 0.176.0
+<!-- Released on: 21 March 2024 -->
+#### New features and enhancements:
```
Comment (brian-f-harness) [inline release-notes/infrastucture-as-code-management.md:41]:
```suggestion
#### New features and enhancements
```

File: release-notes/infrastucture-as-code-management.md [hunk 3]
```diff
@@ -28,4 +28,21 @@ These release notes describe recent changes to Harness Infrastructure as Code Ma
...
+  - **Enhanced Lock Icon Tooltips and Warnings:** We've updated the tooltips and warning messages for the lock icon in Workspace resources to improve clarity and accessibility. (IAC-1577)
+
+### Version: 0.175.0
+<!-- Released on: 15 March 2024 -->
+#### Fixed issues:
```
Comment (brian-f-harness) [inline release-notes/infrastucture-as-code-management.md:46]:
```suggestion
#### Fixed issues
```

File: release-notes/infrastucture-as-code-management.md (hunk 2 above)
Comment (brian-f-harness) [reply release-notes/infrastucture-as-code-management.md:41]: Nit - Recommend removing colons from headings. I think this used to be in the style guide, but I don't see it now. Just for your consideration. Most modules do not include colons for these headings.

File: release-notes/infrastucture-as-code-management.md [hunk 4]
```diff
@@ -28,4 +28,21 @@ These release notes describe recent changes to Harness Infrastructure as Code Ma
...
+## March 2024
+### Version: 0.178.0
+<!-- Released on: 25 March 2024 -->
+#### New features and enhancements:
+  - **Connector Validation Enhancement:** We've implemented a validation check for connectors when creating or editing workspaces. This update ensures that all connectors used are currently available and of the correct type before proceeding with the operation, to help prevent errors related to non-existent or incorrect type connectors, enhancing the reliability and efficiency of workspace management. (IAC-1602)
+
+### Version: 0.176.0
+<!-- Released on: 21 March 2024 -->
+#### New features and enhancements:
+  - **Enhanced Lock Icon Tooltips and Warnings:** We've updated the tooltips and warning messages for the lock icon in Workspace resources to improve clarity and accessibility. (IAC-1577)
```
Comment (brian-f-harness) [inline release-notes/infrastucture-as-code-management.md:42]: If Workspace resources is a UI element, recommend making it bold, if not, please ignore.

https://harness.atlassian.net/wiki/spaces/DOCS/pages/21130772574/Documentation+Style+Guide#UI-elements

File: release-notes/infrastucture-as-code-management.md [hunk 5]
```diff
@@ -28,4 +28,21 @@ These release notes describe recent changes to Harness Infrastructure as Code Ma
...
+
+### Version: 0.175.0
+<!-- Released on: 15 March 2024 -->
+#### Fixed issues:
+  - **Improved Header Visibility:** We resolved the issue of overlapping headers in side-by-side workspace state comparisons, enhancing readability and accessibility for the 'View Execution' and filename headers. (IAC-1573)
```
Comment (brian-f-harness) [inline release-notes/infrastucture-as-code-management.md:47]: If View Execution is a UI element, recommend making it bold, if not, please ignore.

File: release-notes/infrastucture-as-code-management.md [hunk 6]
```diff
@@ -28,4 +28,21 @@ These release notes describe recent changes to Harness Infrastructure as Code Ma
...
+  - **Planned Changes Count Accuracy:** We refined the 'Planned changes' count to include only Added, Changed, and Deleted resources, excluding Unchanged resources, for a more accurate reflection of significant changes. (IAC-1561)
```
Comment (brian-f-harness) [inline release-notes/infrastucture-as-code-management.md:48]: Same here for Planned changes if it's a UI element.

**Review by brian-f-harness — APPROVED** [review]
Looks good. A few optional suggestions for your consideration.

File: release-notes/infrastucture-as-code-management.md (hunk 1 above)
Comment (richardblack-Harness) [reply release-notes/infrastucture-as-code-management.md:36]: Agreed, I'll remove the colon.

File: release-notes/infrastucture-as-code-management.md (hunk 2 above)
Comment (richardblack-Harness) [reply release-notes/infrastucture-as-code-management.md]: Agreed, I'll remove the colons

File: release-notes/infrastucture-as-code-management.md (hunk 5 above)
Comment (richardblack-Harness) [reply release-notes/infrastucture-as-code-management.md]: Fair point

**Review by brian-f-harness — APPROVED** [review]
LGTM

**Pull request merged**